cmd/snapshot-controller/main.go

```diff
@@ -100,11 +103,18 @@ func ensureCustomResourceDefinitionsExist(client *clientset.Clientset) error {
 		return true, nil
 	}
 
-	// with a Factor of 1.5 we wait up to 7.5 seconds (the 10th attempt)
+	maxMs := retryCRDIntervalMax.Milliseconds()
+	if maxMs < 100 {
+		maxMs = 100
```

Comment:
What is the relationship between this 100 ms and Duration of 100 * time.Millisecond on line 115?

Reply:
Explained in the comment, LMK if it's not clear.